packages/opal-server/opal_server/scopes/scope_store.py

```diff
@@ -0,0 +1,56 @@
+from pathlib import Path
+from typing import List
+
+from opal_server.redis import RedisDB
+from opal_server.scopes.pull_engine import CeleryPullEngine
+from opal_common.schemas.scopes import Scope
+
+
+class ScopeNotFound(Exception):
+    pass
+
+
+class ScopeStore:
+    PREFIX = 'io.permit.scope'
```

Comment:
should be configurable from a config var. Maybe an open source user want to change that?

Reply:
the PREFIX denotes the type we save (i'll change it to `opal.scope`)